docker-compose.yml

```diff
@@ -14,30 +14,28 @@ services:
 
   web:
     extends: base
-    command: web -b 0.0.0.0:8000
+    command: web -b 0.0.0.0:8000 --reload
     ports:
       - "8080:8000"
-    links:
-      - mysql
-      - redis
     environment:
-      DEBUG: 1
+      - DEBUG=1
+      - ALLOWED_HOSTS=127.0.0.1,localhost,web
 
   worker:
     extends: base
-    command: worker -l info --queues localhost,celery
-    links:
-      - prometheus
-      - mysql
-      - redis
+    command: worker -l info --queues localhost,celery,prometheus,alertmanager
```

Comment:
Typically the worker only needs to listen to 2 queues at max. 
`celery` is the default queue for almost everything (notifications, etc) and there should be a single named queue for each Prometheus worker that matches is host name. In the case of the docker container it might just be `prometheus` so perhaps this command should be `--queues prometheus,celery`

Reply:
Alright, it seems to work properly with:

```
command: worker -l info --queus prometheus,celery
```